Tools/Scripts/webkitpy/layout_tests/controllers/layout_test_runner_unittest.py

```diff
@@ -327,6 +327,17 @@ def test_shard_every_file(self):
              ('.', ['ietestcenter/Javascript/11.1.5_4-4-c-1.html']),
              ('.', ['dom/html/level2/html/HTMLAnchorElement06.html'])])
 
+    def test_large_shard(self):
```

Comment:
Can we add a second test that actually tests the property I was arguing for before (that we can only split shards or add tests to a (possibly new split) shard)?